<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SideWidgetFileSystem</class>
 <widget class="QWidget" name="SideWidgetFileSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Object Scope &amp; Context Details</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>1</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLineEdit" name="txtCurrentPath">
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTreeView" name="treeView"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="btnBrowse">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Browse for new directory</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/CoreGui.qrc">
       <normaloff>:/qtilities/coregui/icons/file_open_16x16.png</normaloff>:/qtilities/coregui/icons/file_open_16x16.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="btnCdUp">
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Go to parent directory</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/CoreGui.qrc">
       <normaloff>:/qtilities/coregui/icons/up_16x16.png</normaloff>:/qtilities/coregui/icons/up_16x16.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/CoreGui.qrc"/>
 </resources>
 <connections/>
</ui>
